functional_tests/scylla_operator/conftest.py

```diff
@@ -237,3 +237,22 @@ def skip_if_no_tls(request, tester: ScyllaOperatorFunctionalClusterTester) -> No
             and not tester.params.get("k8s_enable_tls")
             and not tester.params.get("k8s_enable_sni")):
         pytest.skip('test requires k8s_enable_tls to be enabled')
+
+
+def pytest_collection_modifyitems(items):
```

Comment:
Docs say it has 3 parameters: https://docs.pytest.org/en/latest/reference/reference.html#collection-hooks

Then, it is unclear what failures are refered to by the commit topic - `fix(test_order): Force test order to reduce failures`

Without this change the test order is also static.
So, it is not "forcing", it is "changing" the order of tests.
Why?

Reply:
Also if your tests are dependant on the test order, you have a fundamental flaw in your tests :)